test/data.jl

```diff
@@ -71,6 +71,28 @@ module TestData
     sdf6f = view(df6, UInt64[1, 2])
     @test size(sdf6f) == (2,3)
 
+    #test_group("getindex")
+    v6a = sdf6a[1]
+    @test isa(v6a, DataArray)
+    @test v6a == df6[1, :][1]
+    v6b = sdf6a[:A]
+    @test isa(v6b, DataArray)
+    @test v6b == df6[1, :][:A]
+    sdf6g = sdf6a[[1, 2]]
+    @test isa(sdf6g, SubDataFrame)
+    @test sdf6g[:] == df6[1, [1, 2]]
+    sdf6h = sdf6a[[:A, :B]]
+    @test isa(sdf6h, SubDataFrame)
+    @test sdf6h[:] == df6[1, [:A, :B]]
+    sdf6i = sdf6a[[:A, :B]]
```

Comment:
I don't think you need `subdf6i`: it's obvious it won't be `=== sdf6h` since you create a new object, and to check that parents are equal you can just use `df6`.